src/components/Contributor.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import DiscordIcon from "../assets/icons/discord-fill.svg";
+import GithubIcon from "../assets/icons/github-fill.svg";
+import LinkedInIcon from "../assets/icons/linkedin-box-fill.svg";
+import ContributorModal from "./ContributorModal";
+
+export default function Contributor(props) {
+  const [showModal, setShowModal] = useState(false);
+
+  const handleShow = () => setShowModal(true);
+  const handleClose = () => setShowModal(false);
+
+  return (
+    <div className="contributor">
```

Comment:
Can this be fixed to be semantically correct, instead of using a whole bunch of divs make it to where it's more accessible